test/parallel/test-fs-chmod.js

```diff
@@ -7,7 +7,7 @@ var got_error = false;
 var success_count = 0;
 var mode_async;
 var mode_sync;
-var is_windows = process.platform === 'win32';
+var is_windows = common.isWindows;
```

Comment:
This one has a couple.. Would be nice if we could make `isWindows` a more consistent name, but meh. (Might be too much of a style change, though that matters less for tests.)